### PR TITLE
Update docs with 'Getting started' section and add fonts guide

### LINKED ISSUE #468: Add a 'Getting started'  section to the main README
Right now we have several places where there is documentation spread around packages. We need a unified 'Getting started' section on the main README with at least the following steps:

- Installing the dependencies
- Adding the `ThemeProvider` and `GlobalStyles`
- Loading fonts
- Including polyfills (such as objectFitPolyfill)
- Removing or moving the no longer relevant [Using with WebPack](https://github.com/Amsterdam/amsterdam-styled-components#using-with-webpack) section.

### PR DESCRIPTION
**Pull request by jonkoops**
Closes #468

### CODE REVIEW

File: packages/asc-assets/README.md [hunk 1]
```diff
@@ -25,29 +25,43 @@ return <IconName />
...
+Add the following code to your `index.html` file:
+
+```html
+<!-- This script is needed as specified by the font license -->
+<script src="https://static.amsterdam.nl/fonts/mtiFontTrackingCode.min.js"></script>
+<link href="https://static.amsterdam.nl/fonts/fonts.css" rel="stylesheet" />
 ```
 
-## Optimized SVGs
+There is nothing more to do, the fonts should not be available in your application.
```
Comment (robinpiets) [inline packages/asc-assets/README.md:42]: There is nothing more to do, the fonts should `not` -> `now` be available in your application.

File: packages/asc-ui/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This is the React implementation of the styled-components
 Add this package to your project by running:
 
 ```bash
-yarn add @datapunt/asc-ui # or npm install @datapunt/asc-ui --save
+npm install add @datapunt/asc-ui
```
Comment (robinpiets) [inline packages/asc-ui/README.md:12]: Typo: npm install **add** @datapunt/asc-ui

File: packages/asc-assets/README.md (hunk 1 above)
Comment (jonkoops) [reply packages/asc-assets/README.md]: Thanks for checking, fixed the typos.

File: packages/asc-ui/README.md (hunk 2 above)
Comment (chrisvanmook) [reply packages/asc-ui/README.md]: isn't this supposed to be `npm install -S @datapunt/asc-ui`?

**Pull request merged**